liblava/resource/generic_mesh.hpp

```diff
@@ -67,7 +67,7 @@ struct generic_mesh_data {
  * @tparam T Input vertex struct
  */
 template<typename T = vertex>
-struct generic_mesh : id_obj {
+struct generic_mesh : entity {
```

Comment:
Ahh this CI error was driving me crazy! It built just fine on my machine, even after re-cloning and trying both Ninja and Make. I couldn't figure it out.

Reply:
Yes, those were the last changes that had to be put on your branch so that the merge would work later. But it's fixed now.